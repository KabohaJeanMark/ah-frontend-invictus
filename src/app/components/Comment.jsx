import React from 'react';
import PropTypes from 'prop-types';
import '../../assets/styles/login.scss';


const Comment = ({ handleSubmit, handleChange }) => {
  return (
    <div>
      <div className="row">
        <div className="col-md-5" />
        <form onSubmit={handleSubmit}>
          <label>Comment</label>
          <textarea id="CommentBody" onChange={handleChange}
            className="form-control input-field" name="commentBody"
            placeholder="Enter your Comments" />
          <input type="submit" className="btn btn-primary login-btn" />
        </form>
      </div>
    </div>
  );
};

Comment.propTypes = {
  handleSubmit: PropTypes.func,
  handleChange: PropTypes.func
};

export default Comment;
