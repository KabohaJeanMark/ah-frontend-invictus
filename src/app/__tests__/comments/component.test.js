import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Comment from '../../components/Comment';

describe('<Comment />', () => {
  let wrapper;
  it('it fetches comments without crashing', () => {
    wrapper = shallow(<Comment />);
  });

  it('matches snapshot', () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
