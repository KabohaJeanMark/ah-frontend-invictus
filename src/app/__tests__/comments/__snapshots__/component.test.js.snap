// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Comment /> matches snapshot 1`] = `
<div>
  <div
    className="row"
  >
    <div
      className="col-md-5"
    />
    <form>
      <label>
        Comment
      </label>
      <textarea
        className="form-control input-field"
        id="CommentBody"
        name="commentBody"
        placeholder="Enter your Comments"
      />
      <input
        className="btn btn-primary login-btn"
        type="submit"
      />
    </form>
  </div>
</div>
`;
